capabilities/display_capabilities.js

```diff
@@ -847,82 +767,7 @@ SDL.templateCapabilities = {
             },
             "numCustomPresetsAvailable": 8
         },
-        "buttonCapabilities": [{
-            "name": "PRESET_0",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_1",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_2",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_3",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_4",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_5",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_6",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_7",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_8",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_9",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "OK",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "SEEKLEFT",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "SEEKRIGHT",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "TUNEUP",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "TUNEDOWN",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }],
+        "buttonCapabilities": SDL.ButtonCapability,
```

Comment:
```suggestion
        "buttonCapabilities": SDL.ButtonCapability.filter(button => button.name !== 'PLAY_PAUSE'),
```
Ah, looks like `PLAY_PAUSE` was not present in this template's capabilities originally

Reply:
Done bc91ea36735350b9397a2a6a8444e89b0bdc2a48